en/peripherals/mavlink_peripherals.md

```diff
@@ -5,31 +5,45 @@ These communication channels are configured using the [MAVLink parameters](../ad
   The default values are:
-  - *Normal*: Standard set of messages for a GCS.
+  - *Normal*: Standard set of messages for a GCS. 
   - *Custom* or *Magic*: Nothing (in the default PX4 implementation).
-    These might be used for testing when developing a new mode.
+     Modes may be used for testing when developing a new mode.
   - *Onboard*: Standard set of messages for a companion computer.
   - *OSD*: Standard set of messages for an OSD system.
   - *Config*: Standard set of messages and rate configuration for a fast link (e.g. USB).
   - *Minimal*: Minimal set of messages for use with a GCS connected on a high latency link.
+  - *ExtVision* or *ExtVisionMin*: Messages for offboard vision systems (ExtVision needed for VIO).
+  - *Iridium*: Messages for an [Iridium satellite communication system](../advanced_features/satcom_roadblock.md).
   
+  > **Note** If you need to find the specific set of message for each mode search for `MAVLINK_MODE_` in [/src/modules/mavlink/mavlink_main.cpp](https://github.com/PX4/Firmware/blob/master/src/modules/mavlink/mavlink_main.cpp).
+
+  <span></span>
   > **Tip** The mode defines the *default* messages and rates.
-  A connected MAVLink system can still request the streams/rates that it wants using [MAV_CMD_SET_MESSAGE_INTERVAL](https://mavlink.io/en/messages/common.html#MAV_CMD_SET_MESSAGE_INTERVAL).
-- [MAV_X_RATE](../advanced_config/parameter_reference.md#MAV_0_MODE) - Set the maximum data rate for this instance (bytes/second). 
-  This is the combined rate for all streams of individual message (the rates for individual messages are reduced if the total rate exceeds this value).
-  The default setting will generally be acceptable, but might be reduced if the telemetry link becomes saturated and too many messages are being dropped.
-In many cases the default should be good however.
-- [MAV_X_FORWARD](../advanced_config/parameter_reference.md#MAV_0_FORWARD) - Enable forwarding of MAVLink packets received by the current instance onto other interfaces. This might be used, for example, to transfer messages between a GCS and a companion computer so that the GCS can talk to a MAVLink enabled camera connected to the companion computer.
+    A connected MAVLink system can still request the streams/rates that it wants using [MAV_CMD_SET_MESSAGE_INTERVAL](https://mavlink.io/en/messages/common.html#MAV_CMD_SET_MESSAGE_INTERVAL).
+- [MAV_X_RATE](../advanced_config/parameter_reference.md#MAV_0_MODE) - Set the maximum *data rate* for this instance (bytes/second).
+  - This is the combined rate for all streams of individual message (the rates for individual messages are reduced if the total rate exceeds this value).
+  - The default setting will generally be acceptable, but might be reduced if the telemetry link becomes saturated and too many messages are being dropped.
+  - A value of 0 sets the data rate to half the theoretical value.

```

Comment:
Note, I'm still clarifying if this is correct with you in https://github.com/PX4/px4_user_guide/pull/508/files#r300223251